Doc/howto/clinic.rst

```diff
@@ -1305,36 +1307,36 @@ will be automatically registered with Argument Clinic; its name
 will be the name of your class with the ``_converter`` suffix
 stripped off.  (This is accomplished with a metaclass.)
 
-You shouldn't subclass ``CConverter.__init__``.  Instead, you should
-write a ``converter_init()`` function.  ``converter_init()``
-always accepts a ``self`` parameter; after that, all additional
+You shouldn't subclass :py:meth:`!CConverter.__init__`.  Instead, you should
+write a :py:meth:`!converter_init` function. :py:meth:`!converter_init`
+always accepts a *self* parameter; after that, all additional
 parameters *must* be keyword-only.  Any arguments passed in to
 the converter in Argument Clinic will be passed along to your
-``converter_init()``.
+:py:meth:`!converter_init`.
 
-There are some additional members of ``CConverter`` you may wish
+There are some additional members of :py:class:`!CConverter` you may wish
 to specify in your subclass.  Here's the current list:
 
-``type``
+:py:attr:`!type`
     The C type to use for this variable.
     ``type`` should be a Python string specifying the type, e.g. ``int``.
     If this is a pointer type, the type string should end with ``' *'``.
 
-``default``
+:py:attr:`!default`
```

Comment:
Why not the following?
```rst
.. attr:: default
```
You need to declare that the current class is `clinic.CConverter`.

Reply:
Ah, of course.